wikimedia/wikipedia-ios
# [ALT TEXT] Add alt ext developer settings toggle

**Phabricator:** https://phabricator.wikimedia.org/T370221

### Notes
* This PR adds the "Enable alt text experiment" option to our developer settings, allowing the user to turn this functionality on/off while it's still under development

### Test Steps
1. On the "About" view, tap the "About" title 7 times
2. The Developer settings should appear; it should display the option for enabling the alt text experiment
3. Enable this option and leave this section. Termite the app
4. The option should remain enabled.
5. Disable, terminate the app
6. Open developer settings again. It should remain disabled
